netjsonconfig/schema.py

```diff
@@ -8,8 +8,11 @@
 
 DEFAULT_FILE_MODE = '0644'
 X509_FILE_MODE = '0600'
+BLANK_PATTERN = '(\s)'
```

Comment:
We are not using `BLANK_PATTERN` anymore. Is it safe to remove this?

Reply:
indeed, removing